displays/cavity_display/frontend/fault_count_display.py

```diff
@@ -59,11 +60,10 @@ def __init__(self, lazy_fault_pvs=True):
         self.start_selector.editingFinished.connect(self.update_plot)
         self.end_selector.editingFinished.connect(self.update_plot)
 
-        # removing for now as POT faults no longer seem to be much of an issue
-        # in terms of distorting the data
-        # TODO: generalize ability to suppress any fault(s)
-        # self.hide_pot_checkbox = QCheckBox(text="Hide POT faults")
-        # self.hide_pot_checkbox.stateChanged.connect(self.update_plot)
+        self.hide_fault_combo_box = QComboBox()
+        tlc_list: List[str] = self.make_tlc_list()
+        self.hide_fault_combo_box.addItems(["Omit a TLC?"] + tlc_list)
```

Comment:
I think it would be helpful to have a text label next to the combo box that makes it obvious what's going on when a fault is selected! So something like "fault to suppress:" with maybe the first option just being blank

Reply:
A fun feature is that the first TLC is the offline fault it is indeed a blank space " "
... Which reminds me that I should make sure the fault suppression actually works for the blank offline fault

Reply:
I can definitely add helpful text! I'm thinking I might change the first message on the combo box to "select a TLC" to avoid the confusion of someone unintentionally selecting the blank and suppressing all offline faults